client/src/components/MainNavbar/MainNavbar.jsx

```diff
@@ -1,47 +1,55 @@
 import React from 'react'
+import { authActions } from '../../actions/auth'
 import { NavLink } from 'react-router-dom'
 import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
-import GoogleAuth from '../GoogleAuth/GoogleAuth'
 import api from '../../apis/backend'
 class MainNavbar extends React.Component {
-  handleLogoutClick = () => {
-    api.logout()
-  }
-
   render() {
     return (
       <nav className="App-header">
         <div className="ui secondary pointing menu">
-          <NavLink className="item" to="/" exact>
-            Home
-          </NavLink>
-          <NavLink className="item" to="/games">
-            Games
-          </NavLink>
+          <div className="ui simple dropdown">
+            <div className="item">
+              <i className="user icon" />
+              Account
+            </div>
+            <div className="menu">
+
+              {api.isLoggedIn() && (
+                <div className="item">
+                  <NavLink className="item" to="/" onClick={this.props.logout}>Logout</NavLink>
+                </div>)}
+
+              {api.isLoggedIn() && (
+                <div className="item">
+                  <NavLink className="item" to="/profile">My Profile</NavLink>
+                </div>
+              )}
+              <div className="item">
+                <NavLink className="item" to="/login">Log In</NavLink>
+              </div>
 
-          <div className="right menu">
-            <NavLink to="/profile">Profile</NavLink>
-            {!api.isLoggedIn() && <NavLink to="/signup">Signup</NavLink>}
-            {!api.isLoggedIn() && <NavLink to="/login">Login</NavLink>}
-            {api.isLoggedIn() && (
-              <NavLink to="/" onClick={this.handleLogoutClick}>
-                Logout
-              </NavLink>
-            )}
-            <GoogleAuth />
+              <div className="item">
+                <NavLink className="item" to="/signup">Sign Up</NavLink>
+              </div>
```

Comment:
Should there also be a conditional check for these too?

`{!api.isLoggedIn() && ...}`